### PR TITLE
Themes

### LINKED ISSUE #105: refactor: convert css-in-js solutions to sass modules
1. Convert existing components

- [x] markdown
- [x] anchor links
- [x] aside
- [x] video
- [ ] left nav
- [ ] main
- [ ] edit
- [ ] container
- [ ] homepage components

2. Distribute sass variables to sass modules globally

### LINKED ISSUE #203: Homepage Callouts not sitting in columns on the grid
image

I'm trying to layout a new component with a gray10 background an it's messing with my layout.

image

### PR DESCRIPTION
**Pull request by vpicone**
This removes the previous css-in-js styles in favor of css modules

closes #105, prepares for #213 

closes #203

### CODE REVIEW

File: packages/gatsby-theme-carbon/src/components/Homepage/Banner.module.scss [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  color: $inverse-01;
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-end;
+  height: 100%;
+}
+
+.row {
+  height: 100%;
+  h1 {
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/Homepage/Banner.module.scss]: Super nitpicky, but can we un-nest these rules, to follow what we've been trying to do across Carbon codebase. Makes readability easier, and easier for future devs when looking and editing.

File: packages/gatsby-theme-carbon/src/components/Homepage/Callout.module.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  font-size: calc(1.25rem + 0 * (100vw - 20rem) / 62);
+}
+
+.secondColumn {
+  @include carbon--type-style('expressive-paragraph-01');
+  padding: 0 1rem;
+  padding-left: 0;
+  width: 100%;
+  font-size: calc(1.75rem + 0.25 * (100vw - 66rem) / 33);
+  @include carbon--breakpoint('md') {
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/Homepage/Callout.module.scss]: can we add a line break above the breakpoint includes for readability

**Comment by alisonjoseph** [comment]
Does this supersede #209 ?

**Comment by vpicone** [comment]
@alisonjoseph yeah I'll make sure to bring your changes in from #209

**Comment by alisonjoseph** [comment]
👍 thanks! and thanks for all the cleanup work here, looks great 😄

File: packages/gatsby-theme-carbon/src/components/Homepage/Callout.module.scss [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  }
+}
+
+.firstColumn {
+  @include carbon--type-style('expressive-heading-03');
+  margin-bottom: $spacing-07;
+}
+
+.secondColumn {
+  @include carbon--type-style('expressive-paragraph-01');
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/Homepage/Callout.module.scss]:
```suggestion
  @include carbon--type-style('expressive-paragraph-01', true);
```

File: packages/gatsby-theme-carbon/src/components/Homepage/Callout.module.scss [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+  max-width: 74.25rem;
+  flex-direction: column;
+
+  @include carbon--breakpoint('md') {
+    flex-direction: row;
+  }
+}
+
+.firstColumn {
+  @include carbon--type-style('expressive-heading-03');
```
Comment (alisonjoseph) [inline packages/gatsby-theme-carbon/src/components/Homepage/Callout.module.scss]:
```suggestion
  @include carbon--type-style('expressive-heading-03, true');
```

**Pull request merged**